src/app/Plans/components/Wizard/PlanWizard.tsx

```diff
@@ -121,16 +114,23 @@ const PlanWizard: React.FunctionComponent<IPlanWizardProps> = ({
   };
 
   return (
-    <Modal isOpen width="95%" showClose={false} hasNoBodyWrapper>
+    <PageSection variant="light">
+      <Breadcrumb className={`${spacing.mbLg} ${spacing.prLg}`}>
+        <BreadcrumbItem>
+          <Link to={`/plans`}>Migration plans</Link>
+        </BreadcrumbItem>
+        <BreadcrumbItem>Plan Wizard</BreadcrumbItem>
+      </Breadcrumb>
       <Wizard
         title="Create a Migration Plan"
         steps={steps}
-        onClose={onClose}
         onNext={onMove}
         onBack={onMove}
         onSubmit={(event) => event.preventDefault()}
+        hideClose={true}
+        cancelButtonText=""
```

Comment:
This hides the Cancel button. @vconzola do we still want the cancel button in place? It could just bring you back to the plans page.

Reply:
I've restored it to help @vconzola review it.